apps/server/src/shared/domain/interface/permission.enum.ts

```diff
@@ -101,6 +101,7 @@ export enum Permission {
 	SCHOOL_PERMISSION_CHANGE = 'SCHOOL_PERMISSION_CHANGE',
 	SCHOOL_PERMISSION_VIEW = 'SCHOOL_PERMISSION_VIEW',
 	SCHOOL_STUDENT_TEAM_MANAGE = 'SCHOOL_STUDENT_TEAM_MANAGE',
+	SCHOOL_TOOL_ADMIN = 'SCHOOL_TOOL_ADMIN',
```

Comment:
Do you require seed data? It don't exists a rolle with this permission

Reply:
this was already done in https://github.com/hpi-schul-cloud/schulcloud-server/pull/3823